main.py

```diff
@@ -5,6 +5,7 @@
 import time
 import board
 import neopixel
+from rainbowio import colorwheel
```

Comment:
this import was missing after the big merge dance before, one reason for the crash